DeployClient/Program.cs

```diff
@@ -132,19 +132,40 @@ static void Main(string[] args)
 
                 if (!API.Install(sessionGuid, out results))
                 {
-                    DateTime abortTime = DateTime.Now.AddMinutes(10);
                     TimeSpan interval = new TimeSpan(0, 0, 0, 2);
+                    Dictionary<string, dynamic> response;
+                    var successfullyReachedApi = false;
+                    DateTime apiNotFoundAbortTime = DateTime.Now.AddSeconds(Options.InstallationStatusTimeout?? 60);
```

Comment:
Can we please improve the whitespace formatting around the null conditional operator.